### PR TITLE
added batch time saving for all sugar recipes

### PR DESCRIPTION
**Pull request by 8ctopus**
#### Summary

```SUMMARY: Content "added batch time saving of 80% > 1 unit for sugar recipes"```

#### Purpose of change

IRL sugar cooking in batch would bring significant time saving, that should be reflected in the game too.

#### Describe the solution

Added batch saving 80% > 1 unit for all 3 sugar recipes without saving.
Updated sweet water > sugar recipe batch saving from 50% > 4 to 50% > 1.

#### Testing

not tested.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
Sounds good to me. if you want this to be more certain to merge/ merge quicker load up your game with this real quick and show what the new crafting times are for batches of 20 compared to making 1. Always good to doublecheck if everything is working as intended.

**Comment by mkikt4743** [comment]
should be a bit slower per batch I think

**Comment by I-am-Erk** [comment]
I don't think batching saves as much time as you imagine here. These processes all appear to rely on boiling off the water, which is going to take longer the more water you have unless you change equipment to have a wider surface area. I'd be surprised if you could save 10% time

File: data/json/recipes/recipe_food.json [hunk 1]
```diff
@@ -3693,6 +3693,7 @@
     "time": "24 m",
     "charges": 10,
     "autolearn": true,
+    "batch_time_factors": [ 80, 1 ],
```
Comment (I-am-Erk) [inline data/json/recipes/recipe_food.json]: See my previous comment: these batch time factors are way way too efficient.

**Comment by 8ctopus** [comment]
I just returned home, I'll revert shortly.

**Comment by 8ctopus** [comment]
Thank you everyone for the feedback provided.

I've updated all sugar recipes to save 20% time for batches instead of the original 80% gain.

It's true that boiling twice the amount of water will take double the time, however as @I-am-Erk suggested, there can still be gains by increasing the contact surface or increasing the surface temperature (my guess).

It's also an option to ignore this merge request, however completely dropping the speed gains would make the recipes not used much in practice because they take way too long.

**Pull request merged**